onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -1180,6 +1180,103 @@ def _where_input_wrangler(
     ),
 )
 
+# Call dir(torch.ops.prims) to get the ops below
+PRIMS_OPS_WITH_OP_INFO = (
+    "abs",
+    "acos",
+    "acosh",
+    "add",
+    "amax",
+    "amin",
+    "as_strided",
+    "as_strided_scatter",
+    "asin",
+    "asinh",
+    "atan",
+    "atan2",
+    "atanh",
+    "bitwise_and",
+    "bitwise_not",
+    "bitwise_or",
+    "bitwise_xor",
+    "cat",
+    "ceil",
+    "clone",
+    "conj",
+    "conj_physical",
+    "cos",
+    "cosh",
+    "digamma",
+    "div",
+    "empty",
+    "empty_permuted",
+    "eq",
+    "erf",
+    "erfc",
+    "exp",
+    "exp2",
+    "expm1",
+    "fill",
+    "floor",
+    "fmax",
+    "fmin",
+    "fmod",
+    "full",
+    "full_like",
+    "gcd",
+    "ge",
+    "gt",
+    "hypot",
+    "igamma",
+    "igammac",
+    "imag",
+    "isfinite",
+    "le",
+    "lgamma",
+    "log",
+    "log10",
+    "log1p",
+    "log2",
+    "lt",
+    "maximum",
+    "minimum",
+    "mul",
+    "ne",
+    "neg",
+    "nextafter",
+    "normal",
+    "pow",
+    "prod",
+    "real",
+    "reciprocal",
+    "remainder",
+    "reshape",
+    "round",
+    "rsqrt",
+    "scalar_tensor",
+    "sign",
+    "signbit",
+    "sin",
+    "sinh",
+    "slice",
+    "sqrt",
+    "squeeze",
+    "sub",
+    "sum",
+    "svd",
+    "tan",
+    "tanh",
+    "transpose",
+    "trunc",
+    "uniform",
+    "var",
+    "where",
+)
+
+for op in PRIMS_OPS_WITH_OP_INFO:
+    # Duplicate opinfo for prim ops. The new names all start with "prims_". E.g. "abs" -> "prims_abs".
+    ops_test_common.duplicate_opinfo_for_prims(OPS_DB, op)
```

Comment:
Probably we don't need to throw the exception of "OpInfo {name} not found in the database.". Or update the list as below which meets the above criteria:

['log', 'normal', 'digamma', 'hypot', 'slice', 'gt', 'asin', 'remainder', 'as_strided_scatter', 'view', 'asinh', 'full', 'scalar_tensor', 'where', 'erf', 'round', 'atan2', 'amax', 'sqrt', 'where', 'tan', 'fmin', 'empty', 'amin', 'real', 'sign', 'div', 'trunc', 'log1p', 'bitwise_and', 'amin', 'trunc', 'rsqrt', 'full', 'bitwise_or', 'lt', 'log10', 'pow', 'cat', 'cosh', 'add', 'sin', 'eq', 'igamma', 'eq', 'asin', 'exp2', 'conj_physical', 'lgamma', 'pow', 'ge', 'acos', 'transpose', 'as_strided', 'full_like', 'gt', 'imag', 'gcd', 'real', 'cosh', 'fmod', 'ge', 'cos', 'minimum', 'sub', 'igammac', 'conj', 'transpose', 'tanh', 'log10', 'tanh', 'le', 'atanh', 'var', 'clone', 'fmax', 'exp', 'atan', 'amax', 'bitwise_not', 'fmod', 'acosh', 'sum', 'floor', 'tan', 'native_batch_norm', 'bitwise_xor', 'sum', 'mul', 'squeeze', 'fill', 'acos', 'log2', 'sign', 'abs', 'neg', 'rsqrt', 'log2', 'floor', 'sub', 'uniform', 'signbit', 'mul', 'sin', 'sinh', 'expm1', 'ceil', 'neg', 'erfc', 'sinh', 'atan2', 'maximum', 'bitwise_not', 'imag', 'le', 'reciprocal', 'abs', 'erfc', 'clone', 'bitwise_xor', 'ne', 'log', 'prod', 'round', 'svd', 'reciprocal', 'exp', 'squeeze', 'atan', 'remainder', 'nextafter', 'lt', 'cos', 'erf', 'rand_like', 'isfinite', 'atanh', 'ceil', 'bitwise_or', 'var', 'var_mean', 'add', 'div', 'expm1', 'log1p', 'ne', 'isfinite', 'sqrt', 'reshape', 'lgamma', 'bitwise_and']

Reply:
Yeah, I updated the list to ensure all of them exist in the op_db. There's also some additional ones in https://github.com/pytorch/pytorch/issues/100223 that are named differently I will need to add.

Reply:
Also it looks like torch-nightly has more infos than 2.0 does. How should we handle this difference?

Reply:
> Also it looks like torch-nightly has more infos than 2.0 does. How should we handle this difference?

If there are more opinfos meet the criteria like above ops I list in torch-nightly, it won't fail our tests so I think we don't need to care about them. We just need regularly check if any new ops in torch latest version meet above criteria and update the list accordingly.

If there are less opinfos meet the criteria like above ops I list in torch-nightly, torch-nightly CI tests will fail. Then we need to update the code to copy the opinfos of missed prims op into extra_opinfo.py file so CI tests pass.

Thoughts?

Reply:
Looks like the newly added one is `empty_permuted`. I removed it for now. If we need it we can copy from torch main to extra_opinfo.py.

I think it's good to keep the "not found in the database" error, otherwise we may think we have something we don't.